tests/shared/src/test/scala/zio/schema/StandardTypeGen.scala

```diff
@@ -55,18 +55,18 @@ object StandardTypeGen {
 
   val anyStandardTypeAndGen: Gen[Random, StandardTypeAndGen[_]] = {
     anyStandardType.map {
-      case typ: StandardType.StringType.type     => typ -> Gen.anyString
+      case typ: StandardType.StringType.type     => typ -> Gen.string
       case typ: StandardType.BoolType.type       => typ -> Gen.boolean
-      case typ: StandardType.ShortType.type      => typ -> Gen.anyShort
-      case typ: StandardType.IntType.type        => typ -> Gen.anyInt
-      case typ: StandardType.LongType.type       => typ -> Gen.anyLong
-      case typ: StandardType.FloatType.type      => typ -> Gen.anyFloat
-      case typ: StandardType.DoubleType.type     => typ -> Gen.anyDouble
-      case typ: StandardType.BinaryType.type     => typ -> Gen.chunkOf(Gen.anyByte)
-      case typ: StandardType.CharType.type       => typ -> Gen.anyASCIIChar
-      case typ: StandardType.UUIDType.type       => typ -> Gen.anyUUID
-      case typ: StandardType.BigDecimalType.type => typ -> javaBigDecimal
-      case typ: StandardType.BigIntegerType.type => typ -> javaBigInt
+      case typ: StandardType.ShortType.type      => typ -> Gen.short
+      case typ: StandardType.IntType.type        => typ -> Gen.int
+      case typ: StandardType.LongType.type       => typ -> Gen.long
+      case typ: StandardType.FloatType.type      => typ -> Gen.float
+      case typ: StandardType.DoubleType.type     => typ -> Gen.double
+      case typ: StandardType.BinaryType.type     => typ -> Gen.chunkOf(Gen.byte)
+      case typ: StandardType.CharType.type       => typ -> Gen.asciiChar
+      case typ: StandardType.UUIDType.type       => typ -> Gen.uuid
+      case typ: StandardType.BigDecimalType.type => typ -> Gen.double.map(d => java.math.BigDecimal.valueOf(d))
```

Comment:
Any reason not to continue to refer to `javaBigDecimal` and `javaBigInt` here and below?

Reply:
I had just blanked on their existence. thanks for picking this up.